### PR TITLE
Fixed supermentor message

### PR DESCRIPTION
**Pull request by ladokp**
to be aligned to the fact that 100 sessions on any track gets you the supermentor status.

@iHiD mentioned on discord: A PR to the website to fix the text would be welcomed! 🙂

### CODE REVIEW

File: app/models/badges/supermentor_badge.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class Badges::SupermentorBadge < Badge
   seed 'Supermentor',
     :legendary,
     'supermentor',
-    'Mentored 100 students on a track'
+    'Mentored 100 students on any track'
```
Comment (iHiD) [inline app/models/badges/supermentor_badge.rb]:
```suggestion
    'Mentored 100 students'
```

**Comment by iHiD** [comment]
Is this actually 100 students, or 100 sessions?

**Comment by ladokp** [comment]
This is a good point indeed! If I remember it right, I got the badge when finishing 100 sessions. I'll have a look to check this. :-)

**Comment by ladokp** [comment]
It's 100 sessions. Let's call it "Finshed 100 mentoring sessions"?

**Comment by iHiD** [comment]
Great. Thanks :)

**Comment by ladokp** [comment]
Looks like a timeout happened:
`Error:
Flows::MentorRegistrationTest#test_registers_to_be_a_mentor:
Net::ReadTimeout: Net::ReadTimeout with "Net::ReadTimeout with #<TCPSocket:(closed)>"
    test/system/flows/mentor_registration_test.rb:26:in `block (2 levels) in <class:MentorRegistrationTest>'
    test/support/capybara_helpers.rb:14:in `use_capybara_host'
    test/system/flows/mentor_registration_test.rb:24:in `block in <class:MentorRegistrationTest>'`

**Comment by ladokp** [comment]
@iHiD Is it possible to redo the checks?

**Review by ErikSchierboom — APPROVED** [review]
LGTM!

**Pull request merged**